model/cluster/get_cluster_settings/operations.smithy

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// 
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+namespace OpenSearch
+
+@externalDocumentation(
+    "OpenSearch Documentation": "https://opensearch.org/docs/latest/opensearch/rest-api/cluster-settings/"
+)
+
+@suppress(["HttpUriConflict"])
+@http(method: "GET", uri: "/_cluster/settings")
+@documentation("Returns cluster settings.")
+operation GETClusterSettings {
```

Comment:
minor: can we change "GET" to "Get" in all operation / structure names?

Reply:
Done in new commit.